### PR TITLE
Change terminology: deep sleep, power down and wake to low/full power

### PR DESCRIPTION
**Pull request by martinwork**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,62 @@
...
 })
 ```
 
-You can also ask the micro:bit to enter a ``||power.deepSleep||`` where it will pause until a wake up event occurs and power down at the next opportunity.
+You can send ``||power.lowPowerRequest(LowPowerMode.Wait)||``. Then micro:bit will also pause until a full power event occurs.
+
+The ``||power.lowPowerPause(ms)||`` block will ask the micro:bit to sleep for a set interval in milliseconds.
 
-The ``||power.deepSleepPause(ms)||`` block will also ask the micro:bit to sleep for a set interval in milliseconds.
+In low power mode, micro:bit is asleep, and your program is paused. When micro:bit wakes up to full power mode, your program continues from the point it stopped.
```
Comment (microbit-mark) [inline README.md:24]: I think we usually use the definite article to describe 'the micro:bit'
```suggestion
In low power mode, the micro:bit is asleep, and your program is paused. When the micro:bit wakes up to full power mode, your program continues from the point it stopped.
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,62 @@
...
     led.unplot(2, 1)
-    power.powerDownEnable(PowerDown.allow)
-    power.powerDownRequest()
+    power.lowPowerEnable(PowerDown.allow)
+    power.lowPowerRequest()
 })
 ```
 
-### Wake the micro:bit from sleep ⏰
+### Wake up micro:bit to full power mode ⏰
```
Comment (microbit-mark) [inline README.md:42]:
```suggestion
### Wake up the micro:bit to full power mode ⏰
```

File: README.md [hunk 3]
```diff
@@ -1,58 +1,62 @@
...
 
-In order to wake the micro:bit, you need to define an event to trigger the wake up call.
+In order to wake up micro:bit to full power mode, you need to define an event to trigger the wake up call.
```
Comment (microbit-mark) [inline README.md:44]:
```suggestion
In order to wake up the micro:bit to full power mode, you need to define an event to trigger the wake up call.
```

**Review by microbit-mark — APPROVED** [review]
Aside from suggested changes to the readme, this looks good to me. Thanks @martinwork

**Comment by martinwork** [comment]
Thanks @microbit-mark

**Pull request merged**